models/index.js

```diff
@@ -17,20 +20,34 @@ const {
   DB: dbTitle,
   DB_USER: dbUser,
   DB_PASS: dbPass,
-  DB_HOST: dbHost,
-  JAWSDB_URL: dbJawsUrl
+  DB_HOST: dbHost
 } = process.env;
 
+// Setting options for db connections
```

Comment:
Can this comment be changed to state 'Setting options for db connections for use in session storage'
Just so it's clear that it's used in express-mysql-session

Reply:
Done